main/internal/implementations/terraform_workspace/factory.go

```diff
@@ -24,5 +28,21 @@ func (f *Factory) Instantiate(ctx context.Context, environment string, dragonDro
 // bootstrappedTerraformWorkspace creates a complete implementation of the interfaces.TerraformWorkspace interface with
 // configuration specified via environment variables.
 func (f *Factory) bootstrappedTerraformWorkspace(ctx context.Context, dragonDrop interfaces.DragonDrop, config TerraformCloudConfig) (interfaces.TerraformWorkspace, error) {
-	return NewTerraformCloud(ctx, config, dragonDrop), nil
+	var backendConfig ContainerBackendConfig
+	err := envconfig.Process("DRAGONDROP", &backendConfig)
+	if err != nil {
+		log.Errorf("[cannot create terraform workspace config]%s", err.Error())
+		return nil, fmt.Errorf("[cannot create terraform workspace config]%w", err)
+	}
+
+	switch config.StateBackend {
+	case "s3":
+		return NewS3Backend(ctx, backendConfig, dragonDrop), nil
+	case "azure":
```

Comment:
```suggestion
	case "azurerm":
```
To keep things consistent with how Terraform calls the backend. https://developer.hashicorp.com/terraform/language/settings/backends/azurerm